src/main/java/com/sonymobile/jenkins/plugins/lenientshutdown/PluginImpl.java

```diff
@@ -66,7 +66,7 @@ public class PluginImpl extends Plugin {
     /**
      * Node name -> Set of queue item ids that are allowed to build
      */
-    private Map<String, Set<Long>> permittedSlaveQueuedItemIds = Collections.synchronizedMap(
+    private Map<String, Set<Long>> permittedNodeQueuedItemIds = Collections.synchronizedMap(
```

Comment:
I am trying to remember if there is anything stored to disk in this plugin. This field is not marked as `transient` so it should be stored to disk.
Have you tested that case? Because of the rename, if an old configuration is on disk and tries to load after an update it will fail since the field mentioned on disk is no longer there.